h/session.py

```diff
@@ -126,7 +127,7 @@ def includeme(config):  # pragma: no cover
         #
         # To avoid this we make sure that the lifetime of CSRF tokens is always
         # longer than the lifetimes of auth cookies.
-        timeout=HTML_AUTHCOOKIE_MAX_AGE + 3600,
+        timeout=int(timedelta(hours=1).total_seconds()),
```

Comment:
Shouldn't this be `HTML_AUTHCOOKIE_MAX_AGE + ...`?

Reply:
Yeah, pylint caught that as well. The fact that `session_test.py` didn't catch this suggests that the unit tests aren't as good as they could be